### ISSUE TITLE
linter: import/extensions false positive error with custom aliases

### ISSUE THREAD

**Issue by web-dev-sam** [oxc-project/oxc#23223]
### What version of Oxlint are you using?

1.69.0

### What command did you run?

pnpm dlx oxlint ./resources/js --fix -A unicorn/no-abusive-eslint-disable

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```jsonc
{
    "plugins": [
        "import",
        "vue",
        "typescript",
        "unicorn"
    ],
    "options": {
        "typeAware": true
    },
    "categories": {
        "correctness": "error",
        "suspicious": "error",
        "perf": "warn"
    },
    "rules": {
        "import/extensions": [
            "error",
            "never",
            {
                "vue": "always",
                "css": "always",
                "json": "always",
                "ignorePackages": true
            }
        ],
    }
}
```

### What happened?

There is the extension `.vue` in the import, but its complaining still. I assume because weird aliases like `@cp/` aren't handled probably.

<img width="830" height="134" alt="Image" src="https://github.com/user-attachments/assets/ef08c349-0133-4d97-9425-50e9208cd3e7" />

In my tsconfig:

```json
{
    "compilerOptions": {
        "paths": {
            "@/*": [
                "./resources/js/*"
            ],
            "@cp/*": [
                "./resources/js/components/component-xxxx/*"
            ]
        }
    }
}
```

**Comment by morgan-coded** [comment]
I can take a look at this if it’s still available. The repro looks scoped to `import/extensions` dropping the written extension for a scoped alias import that is explicitly configured.
